requirements/tests.txt

```diff
@@ -0,0 +1 @@
+mock
```

Comment:
As this is required only for Python 2.7, you can include an environment marker, so it won't be installed in newer Python versions (https://www.python.org/dev/peps/pep-0508/#environment-markers)